api/v3/Email/Send.php

```diff
@@ -162,7 +162,8 @@ function civicrm_api3_email_send($params) {
     }
 
     // Change the contact array to the format the hook expects.
-    $contactHookArray = CRM_Utils_Token::getTokenDetails([$contact['contact_id']])[0];
+    // Ensure we get the tokens even if the email address is set to On Hold (this is not relevant for token processing)
+    $contactHookArray = CRM_Utils_Token::getTokenDetails([$contact['contact_id']], NULL, FALSE)[0];
```

Comment:
I am of the view that we should disable the deceased filter too, since these crashes can cause major problems, and [line 151](https://github.com/australiangreens/org.civicoop.emailapi/pull/6/files#diff-d1c70eeb386beae24e3cc3a629071ab92ec88a15cefda8580e81f6ac1182df6bR151) already ensures that we will never send an email if the recipient is deceased.

```suggestion
    $contactHookArray = CRM_Utils_Token::getTokenDetails([$contact['contact_id']], NULL, FALSE, FALSE)[0];
```